core/local/change.js

```diff
@@ -478,6 +489,7 @@ function includeChangeEventIntoFileMove (sameInodeChange /*: ?LocalChange */, e
   log.debug({path: e.path}, 'FileMove + change')
   moveChange.md5sum = moveChange.old.md5sum || moveChange.md5sum
   moveChange.update = _.defaults({
+    md5sum: e.md5sum,
```

Comment:
Don't we already get `md5sum` from `e` via the call to `_.defaults`?

Reply:
Probably :man_facepalming:

Reply:
Fixed